general/1_terminal/Danylo_Kulbachinskiy/script3.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+f="$(date "+%d")"
+arr=("субота" "неділя")
```

Comment:
Let's make the script more portable. Currently, this only works if your machine has Ukrainian language installed.

Unfortunately, it's not on every machine. And neither is English, so let's try to think of a solution that doesn't compare against strings.